test/fuzz/fuzz-netdev-parser/directives.netdev

```diff
@@ -219,3 +219,13 @@ InterfaceId=
 [BareUDP]
 DestinationPort=
 EtherType=
+[Batadv]
```

Comment:
shouldn#t the section name be BatmanAdv? We try to avoid too opaque acronyms, and given that batadv is already an abbreviation of an acronym things get needlessly confusing. There's no shame in typing out [BatmanAdv]

Reply:
[BatmanAdvanced] might be an even better name